telegram/dispatcher.py

```diff
@@ -33,7 +34,8 @@
 logging.getLogger(__name__).addHandler(H)
 
 semaphore = None
-running_async = 0
+async_threads = set()
+""":type: set[Thread]"""
```

Comment:
in order not to leave this "in the air"
it was agreed that we can use the Sphinx docstring comment style for this kind of case (i.e. "standalone" variable)
this was the most reliable form we found and the most friendly to pycharm